ANMN/NRS_AIMS/REALTIME/dest_path.py

```diff
@@ -1,99 +1,31 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3.5
```

Comment:
Unless it is necessary should we just use the default python version?

Reply:
this was a suggestion from @lwgordonimos to do it this way if I understood correctly what the man meant at that time

Reply:
Doing it like these means scripts can be ported over one by one, rather than doing a big bang approach and updating the system interpreter for *everything*.

Also, doesn't need to be specific version, just `/usr/bin/env python3` is fine.